src/components/MovieList/Cards.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { useNavigate } from "react-router-dom";
+
+import {
+  CardStyle,
+  CardsStyle,
+  ButtonStyle,
+  TitleStyle,
+  CardsImageStyle,
+} from "../Styles/Card.style";
+
+const Cards = ({ data }) => {
+  const navigate = useNavigate();
+  const IMG_URL = "https://image.tmdb.org/t/p/w500";
+  const fake_img = "http://via.placeholder.com/1080x1580";
```

Comment:
Extract these constants inside separate folder called constants and import it from there.